pkg/sqlite/migrations/004_cascade_delete_test.go

```diff
@@ -0,0 +1,272 @@
+package migrations_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/operator-framework/operator-registry/pkg/sqlite/migrations"
+)
+
+func TestCascadeDeleteUp(t *testing.T) {
+	// migrate up to, but not including, this migration
+	db, migrator, cleanup := CreateTestDbAt(t, migrations.CascadeDeleteMigrationKey-1)
+	defer cleanup()
+
```

Comment:
you should validate that the table doesn't have the cascades here, before migrating up